package.json

```diff
@@ -60,7 +60,24 @@
                     "type": "string",
                     "default": "vale",
                     "description": "Absolute path to Vale binary. Special var ${workspaceFolder} can be used in path (NOTE: in windows you can use '/' and can omit '.cmd' in path)"
-                }
+                },
+                "vscode-vale.fileExtensions": {
+                    "scope": "resource",
+                    "type": "array",
+                    "default": [
```

Comment:
Does it work by specifying the defaults only here and not in the getter?

Reply:
I don't really understand the question sorry. I pretty much followed the same steps as in the project you linked elsewhere and it seemed to work well and tidily.

Reply:
I was referring to the `get('fileExtensions')` part, where you can also pass a default value.

<img width="444" alt="image" src="https://user-images.githubusercontent.com/1110551/69611419-02d0cd80-102e-11ea-82a5-bde12bc24f25.png">

But I suppose this works too.